tests/unit/test_analytics.py

```diff
@@ -88,7 +86,74 @@ def test_find_periods(
         (dummy_datetime_list[9], dummy_datetime_list[9]),
     ]
 
-    result = sleep_detection._find_periods(window_measurement)
+    result = analytics._find_periods(window_measurement)
+
+    assert result == expected_result, f"Expected {expected_result}, but got {result}"
+
+
+@pytest.mark.parametrize(
+    "non_wear_array, expected_result",
+    [
+        (np.array([0, 1, 1, 1, 0]), []),
+        (np.array([0, 0, 0, 1, 1]), []),
+        (np.array([0, 0, 1, 0, 0]), []),
```

Comment:
We discussed it in private but just adding this comment as a reminder to maybe change the wording of the docs below to better reflect these tests, and also change the third array so that the non wear is at n = 3.